src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+
+  const countButtons = Math.ceil(total / perPage);
+  const buttons: number[] = Array.from(
+    { length: countButtons },
+    (_, i) => i + 1,
+  );
+  const isActive = (pageNumber: number): boolean => {
+    return pageNumber === currentPage;
+  };
+
+  const disableNext = currentPage === 1;
```

Comment:
The logic for disabling the 'Next' button is incorrect. It should be disabled when the current page is the last page, not the first. Swap the conditions for `disableNext` and `disablePrev`.